compiler/passes/flattenFunctions.cpp

```diff
@@ -27,6 +27,7 @@
 #include "stlUtil.h"
 
 static void flattenNestedFunctions(Vec<FnSymbol*>& nestedFunctions);
+static void markTaskFunctionsInIterators(Vec<FnSymbol*>& nestedFunctions);
```

Comment:
Geez, so much stuff will go away when we reverse the introduction of task functions early in the compiler!